src/app/file-manager/warning-modal/warning-modal.component.html

```diff
@@ -0,0 +1,30 @@
+<h2 mat-dialog-title>
+  <i class="fa fa-exclamation-triangle icon-warning" aria-hidden="true"></i>
+  Are you sure you want to
+  <span *ngIf="data == 'exit'"> leave?</span>
+  <span *ngIf="data == 'logout'"> Sign out?</span>
+  <span *ngIf="data == 'cancelAllUploads' || data == 'cancelAllDownloads'"> cancel?</span>
+</h2>
+
+<mat-dialog-content>
+  <div class="flex-container">
+    <p>All current
+      <span *ngIf="data == 'exit' || data == 'logout'">transfers</span>
+      <span *ngIf="data == 'cancelAllUploads'">Uploads</span>
+      <span *ngIf="data == 'cancelAllDownloads'">Downloads</span>
+      will be aborted.
```

Comment:
Instead of `aborted` this should use wording that matches the wording in the UI. In other places we call this operation `Cancel` so it should be `we be canceled.`.